diagnostic.md

```diff
@@ -7,36 +7,37 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:callmelife/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+Create a branch
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git checkout -b response
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git diff path/to/workingDirectory/or/file
```

Comment:
"git diff" is a great tool for comparing where you are currently and a previous commit, or another branch. "git diff" shows you the actually differences in the code. However, in this case, the code we just wrote is all the change that have been made, the easiest way to get a summary would be with "git status".